memory/index.py

```
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from typing import List
+from llama_index.core import Document, StorageContext, VectorStoreIndex, load_index_from_storage
+from llama_index.embeddings.openai import OpenAIEmbedding
+from tools.health_schema import SymptomLog
+from db.health_db import get_entries  # implement/adjust import to your Phase-2 CRUD
+import os, json, pathlib
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import

(F401)